indexer/osm_editor.cpp

```diff
@@ -449,12 +462,20 @@ Editor::SaveResult Editor::SaveEditedFeature(EditableMapObject const & emo)
   {
     ASSERT_NOT_EQUAL(featureStatus, FeatureStatus::Deleted, ("Unexpected feature status."));
 
+    auto const originalFeaturePtr = m_getOriginalFeatureFn(fid);
+    if (!originalFeaturePtr)
+    {
+      LOG(LERROR, ("A feature appears missing/deleted while saving."));
+      alohalytics::LogEvent("Editor_MissingFeature_Error");
```

Comment:
This is the first time I see this event - do we have similar events? Is it processed?